Docs 9549 - Updates for editorial review (#106)
mulesoft/docs-partner-manager: modules/ROOT/pages/create-outbound-message-flow.adoc

= Create and Configure an Outbound Message Flow

> Create an outbound message flow to receive application messages from your backend systems, transform, and then send the translated EDI message to your trading partners.

Reviewer: What are the actual steps here?
receive + transform (or translate?) + send?
Clarify, and avoid the wordiness around inbound and outbound, as suggested earlier: GLOBAL

Author: Done

After you create the outbound message flow, you configure the receive endpoint, source message type, target message type, and target endpoint.

> == Create an Outbound Message Flow

Reviewer: Ditto all edits made to the Inbound Message Flow peer topic.
The wording and organization into substeps, etc., should be almost exactly the same.
I think we are breaking this down too granularly.

Reviewer: Proofed only the rest of this topic set.

> . From your Anypoint Partner Manager *Sandbox* or *Production* environment, select *Message Flows* in the left navigation menu.

Reviewer: We are referring to the environment in general, not the UI section label, so...:
```suggestion
. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the left navigation menu.
```
or name the UI labels and do not call them "environments."

. Expand the *Source at <host-name>* section.
. In *Endpoint:Source at <host-name>*, click *Select*.

> . To use an existing receiving endpoint:

Reviewer: receiving or receive? Search through topics and standardize.

Author: "Receive"  to match UI - fixed

.. Select the endpoint from the list and click *Select* in the lower right side of the page. +
The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
.. Click *Add* to add a unique identifier for the endpoint.
. To create a new receiving endpoint:

> .. Click *New* on the right of the search box.

Reviewer: in the right area of the search box or TO the right of the search box?

Author: Done

> .. From the *Owner* drop-down, select the endpoint's owner.

Reviewer: ```suggestion
.. From the *Owner* drop-down list, select the endpoint's owner.
```
If I were to omit any word here, it would be "drop-down" rather than "list."

.. From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
* xref:endpoint-https-send.adoc[*HTTP(S)*]
* xref:endpoint-sftp-send.adoc[*SFTP*]
. Click *Save*. +
The endpoint is added to the message flow.

== Configure the Source Message Type

. Expand the *Source message type* section, and click *Select*.

> . To use an existing message type, select the message type from the list and click *Select* in the lower right side of the page. +

Reviewer: Here is yet a third treatment of these two options:
-"alternatively" plain text in one step
-substeps for each option
-separate steps for each option
Please pick one treatment and standardize to it throughout.
Consider text between title and Step 1 that prepares audience for either creating a new or editing an existing, throughout.

The message type is added to the message flow.
. To create a new message type:
.. Click *New* on the right side of the search box.
.. From *Format*, select either:
* *JSON*
* *XML*
.. From *Schema*, click *Choose file* and browse to the location of your custom schema.
.. Accept the default value for *Message type identifier* or enter a new value.

> .. Click *Choose file* to browse to the location of the DataWeave map, and select it.

Reviewer: ```suggestion
.. Click *Choose file* to browse to the location of the DataWeave map and select it.
```

.. Click *Save*.
. Under the *Reference IDs* section:
.. Click *Select* to add the reference identifier for the trading partner.
.. From the *<host-name> reference ID* drop-down list, select your organization’s internal reference ID (such as business unit name) as it would appear in the data element of the source JSON or XML payload from your backend systems.

== Configure the Target Message Type

. Expand the *Message type* section.
. Click *Select* to select a message type or to create a new message type.

> . To use an existing message type, select the message type from the list and click *Select*.
> . To create a new message type:

Reviewer: s/b substeps to previous
```suggestion
. To use an existing message type, select the message type from the list and click *Select*:
.. To create a new message type:
```

> .. Click *New* on the right of the search box.

Reviewer: ditto "on" vs."to"

Author: Done

.. From *Format*, select *X12*.
.. From *Version*, select the X12 version.
.. From *Message type*, select the message type.
.. From schema select *Use standard X12-<version>-<message-type>* or select *Customize* to define a custom schema.
.. Click *Save*. +
The message type and its validation settings are added to the message flow.

> . In the *Acknowledgment* section, select the *Expect functional acknowledgment* checkbox if you expect to receive functional acknowledgments (997) back from your trading partner.

Reviewer: ```suggestion
. In the *Acknowledgment* section, select *Expect functional acknowledgment* if you expect your partner to return functional acknowledgments (997).
```
